pilot/pkg/serviceregistry/external/controller_test.go

```diff
@@ -1,383 +0,0 @@
-// Copyright 2017 Istio Authors
```

Comment:
Rather than a separate file I made this integrated into the other tests so that we test a changed, and assert we get the right instances, right xds, etc